### ISSUE TITLE
Reference to non-existing wiki page

### ISSUE THREAD

**Issue by ehrenfeu** [scijava/scijava-scripts#26]
This page doesn't seem to exist, and I failed to find it anywhere else:

https://github.com/scijava/scijava-scripts/blob/818b9717916d4fc62c3d7c8098448a7b2f0d5f6e/release-version.sh#L26

**Comment by ctrueden** [comment]
The GPG-Signing page used to exist, but it became obsolete after we switched from Jenkins to Travis CI. The information on it was so outdated and misleading that I decided to delete the page rather than updating the information. You no longer need any GPG configuration locally in order to run the `release-version.sh` script; it's Travis CI that needs it.

As an aside: you _can_ look at deleted wiki pages, by cloning the wiki repository and digging through the Git history. (But it's not currently possible to browse it online; see isaacs/github#1265).

I made a commit removing this obsolete link: 8372074990d14f5a6372608547c0dbd8f5cfb4c6.

@ehrenfeu I'm happy to answer any questions you have about the GPG signing; just ask. But it's a hairy process and I already lost the link to the guides I followed when adding GPG signing support to the `travisify.sh` and `travis-build.sh` scripts.
